Define conflicts with Solaris regset.h; should either always undef first,
or upstream should rename their #define.

file upstream bug and determine desired fix.

--- a/Source/ThirdParty/ANGLE/src/common/debug.h	2018-02-14 09:25:23.653561020 -0800
+++ b/Source/ThirdParty/ANGLE/src/common/debug.h	2018-02-14 09:27:14.110989611 -0800
@@ -204,6 +204,7 @@
 #endif
 
 #define WARN() ANGLE_LOG(WARN)
+#undef ERR
 #define ERR() ANGLE_LOG(ERR)
 
 // A macro to log a performance event around a scope.
